hi-ml-azure/src/health_azure/himl.py

```diff
@@ -584,26 +766,35 @@ def _write_run_recovery_file(run: Run) -> None:
 
 
 def convert_himl_to_azureml_datasets(
-        cleaned_input_datasets: List[DatasetConfig],
-        cleaned_output_datasets: List[DatasetConfig],
-        workspace: Workspace) -> Tuple[Dict[str, DatasetConsumptionConfig], Dict[str, OutputFileDatasetConfig]]:
+    cleaned_input_datasets: List[DatasetConfig],
+    cleaned_output_datasets: List[DatasetConfig],
+    workspace: Workspace,
+    strictly_aml_v1: bool
+) -> Tuple[Dict[str, DatasetConsumptionConfig], Dict[str, OutputFileDatasetConfig]]:
     """
     Convert the cleaned input and output datasets into dictionaries of DatasetConsumptionConfigs for use in AzureML.
 
     :param cleaned_input_datasets: The list of input DatasetConfigs
     :param cleaned_output_datasets: The list of output DatasetConfigs
     :param workspace: The AzureML workspace
+    :param ml_client: An optional Azure MLClient object for interacting with Azure resources.
```

Comment:
wrong param? strictly_aml_v1 needs to be added instead

Reply:
True! Fixed